src/it/java/teammates/it/sqllogic/core/DataBundleLogicIT.java

```diff
@@ -233,4 +235,34 @@ public void testPersistDataBundle_typicalValues_persistedToDbCorrectly() throws
         // TODO: incomplete
     }
 
+    @Test
```

Comment:
i think it would be better here if we verify that they are present in the database first, before deleting it